### PR TITLE
feat: add watch later and like feature

### PR DESCRIPTION
**Pull request by jalajcodes**
Live: https://max-video.herokuapp.com

### CODE REVIEW

File: frontend/src/context/likesContext.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        `Video already ${type === "likes" ? "liked" : "disliked"}`
+      );
+    }
+
+    const newStateData = {
+      ...state.data,
+      [type]: [...currentData, video],
+    };
+    console.log(
+      "🚀 ~ file: likesContext.js ~ line 39 ~ addToLikes ~ newStateData",
```
Comment (MayankKumar10) [inline frontend/src/context/likesContext.js]: remove console log

**Review by MayankKumar10 — APPROVED** [review]
Just a suggestion everything else looks good to me you can merge it.

File: frontend/src/context/likesContext.js (hunk 1 above)
Comment (jalajcodes) [reply frontend/src/context/likesContext.js]: thanks for pointing out

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me! Code looks clean and understandable

**Pull request merged**